src/qml/Mysthea/Pages/GameSetup.qml

```diff
@@ -39,18 +41,34 @@ Page {
         }
 
         Repeater {
+            id: repeater
             model: GameSetupModel {
+                id: _gameSetupModel
+                width: root.availableWidth
             }
-            Loader {
-                asynchronous: true
-                active: SwipeView.isCurrentItem || SwipeView.isNextItem
-                        || SwipeView.isPreviousItem
-                sourceComponent: GameSetupStep {
-                    title: model.title
-                    mainImageSource: model.image
-                    content: model.content
-                }
-            }
+        }
+    }
+
+    footer: Pane {
+        anchors.bottom: parent.bottom
```

Comment:
`anchors` here is not necessary, a `Component` set as `footer` is automatically positioned bottom.